src/Language/PureScript/Constants.hs

```diff
@@ -204,6 +204,15 @@ unit = "unit"
 
 -- Core lib values
 
+untilE :: forall a. (IsString a) => a
+untilE = "untilE"
+
+whileE :: forall a. (IsString a) => a
+whileE = "whileE"
+
+forE :: forall a. (IsString a) => a
+forE = "forE"
+
```

Comment:
I moved `untilE` and `whileE` to the `EffectDictionaries` data type so that the implementations can be shared between Effect and ST. The same can be done with `forE`, which is `for` for ST.

Reply:
done